torchgeo/datasets/geo.py

```diff
@@ -487,22 +536,54 @@ def _merge_files(
 
         Returns:
             image/mask at that index
+
+        .. versionchanged:: 0.5
+            Given the *as_time_series* parameter merge files to a single time instance
+            as previously or return a time series for the geographical location
         """
         if self.cache:
             vrt_fhs = [self._cached_load_warp_file(fp) for fp in filepaths]
         else:
             vrt_fhs = [self._load_warp_file(fp) for fp in filepaths]
 
         bounds = (query.minx, query.miny, query.maxx, query.maxy)
-        dest, _ = rasterio.merge.merge(vrt_fhs, bounds, self.res, indexes=band_indexes)
+        if self.as_time_series:
```

Comment:
I think we briefly discussed this @adamjstewart , in my thinking this extra step is necessary but maybe you disagree with this.